src/main/kotlin/com/team4099/robot2021/subsystems/Wheel.kt

```diff
@@ -109,6 +110,18 @@ class Wheel(private val directionSpark: CANSparkMax, private val driveSpark: CAN
     directionSetPoint = direction + directionDifference
   }
 
+  fun setVelocitySetpoint(velocity: LinearVelocity, acceleration: LinearAcceleration) {
+    drivePID.setReference(
+      driveSensor.velocityToRawUnits(velocity),
+      ControlType.kVelocity,
+      0,
+      (Constants.Drivetrain.PID.DRIVE_KS +
```

Comment:
oops the example doesn't use the sign() thing from the actual equation, we need that since static friction is opposite to the direction of motion so ks shouldn't always be positive